tests/context-info.sh

```diff
@@ -0,0 +1,400 @@
+#!/bin/sh
+
+# context-info.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2012-2020, ARM Limited, All Rights Reserved
+#
+# This program is intended for testing the ssl_cntext_info program
+#
+
+set -u
+
+# Go to directory with this script
```

Comment:
I don't think that it's necessary to comment on a "cd" line :)

Reply:
Ok. removed